pkg/sysinfo/meminfo.go

```diff
@@ -0,0 +1,24 @@
+package sysinfo
```

Comment:
```suggestion
package sysinfo // import "github.com/docker/docker/pkg/sysinfo"
```

Reply:
Ah, probably should've mentioned that; so these import comments act at the _package_ level; so technically, only a single file in the package needs to have it to enforce the import.

As we're planning to move to go modules (and thus rename the module), I'm preemptively removing these comments in files I move to other packages (if there's already at least one of those comments in the package) to reduce further code-churn a bit.

Overall the import comments are starting to lose value, as in many cases (if go mod is involved in any way), they're completely ignored.